libs/components/filter-bar/src/lib/modules/filter-bar/filter-bar.component.ts

```diff
@@ -47,17 +52,32 @@ export class SkyFilterBarComponent {
     return filters?.some((filter) => !!filter.filterValue);
   });
 
+  readonly #confirmSvc = inject(SkyConfirmService);
   readonly #modalSvc = inject(SkySelectionModalService);
   readonly #resourceSvc = inject(SkyLibResourcesService);
 
   #filterPickerDescriptor = 'filters';
+  #clearFiltersConfirmTitle = 'Title';
+  #clearFiltersConfirmBody = 'Body';
+  #clearFiltersConfirmApply = 'Clear values';
+  #clearFiltersConfirmCancel = 'Cancel';
 
   constructor() {
     this.#resourceSvc
-      .getString('skyux_filter_bar_filter_picker_descriptor')
+      .getStrings({
+        descriptor: 'skyux_filter_bar_filter_picker_descriptor',
+        title: 'skyux_filter_bar_clear_filters_confirm_title',
+        body: 'skyux_filter_bar_clear_filters_confirm_body',
+        save: 'skyux_filter_bar_clear_filters_confirm_apply_label',
+        cancel: 'skyux_filter_bar_clear_filters_confirm_cancel_label',
+      })
       .pipe(take(1))
```

Comment:
This subscription will not respond to changes to the locale. Could you convert the subscription to a signal and use it that way?
```
readonly #strings = toSignal(
  this.#resourceSvc.getStrings({
    descriptor: 'skyux_filter_bar_filter_picker_descriptor',
    title: 'skyux_filter_bar_clear_filters_confirm_title',
    body: 'skyux_filter_bar_clear_filters_confirm_body',
    save: 'skyux_filter_bar_clear_filters_confirm_apply_label',
    cancel: 'skyux_filter_bar_clear_filters_confirm_cancel_label',
  }),
);
```

```
const strings = this.#strings();

if (!strings) {
  return;
}

const { body, descriptor, message, ... } = strings;

const config: SkyConfirmConfig = {
  message,
  body,
  ...
};
```

Reply:
updated